### DISCUSSION TITLE
Water Sprays in FDS

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12693] [General]
I am trying to read through the FDS documentation and am trying to figure out the following: 

1. Does FDS accurately model water evaporation due to heat flux? (i.e., if I have a nozzle spraying water next to a fire, will the water evaporate due to heat transfer from the fire)

2. Is there a good way to track what percentage of the water evaporates/what percentage remains a liquid? And secondarily, is it possible to track pool depth of water that does not evaporate?

**Comment by mcgratta** [comment]
1. Water droplets absorb radiative and convective heat.
2. Best way to see how much water evaporates is to collect the water on the floor and compare to spray rate.

**Reply by Tbuttts21** [reply]
Thank you for your quick and helpful responses as always.

**Comment by mcgratta** [comment]
Look through the User and Verification Guides for examples.

**Comment by drjfloyd** [comment]
Alsio the validation guide has various experiments with evaporation of droplets.
